src/util/fetch.js

```diff
@@ -1,15 +1,15 @@
 export const fetchJson = (url, options = {}) => {
-    const requestHeaders = new Headers({
+    const requestHeaders = options.headers || new Headers({
```

Comment:
The problem is that adding it by default removes the possibility to disable it. This should be up to the user IMO